api/src/test/java/jakarta/el/TesterUtil.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2025 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+package jakarta.el;
+
+public class TesterUtil {
+
+    public static enum Numbers {
```

Comment:
https://docs.pmd-code.org/latest/pmd_rules_java_codestyle.html#unnecessarymodifier
> Nested enums are automatically `static`.

Reply:
Fixed. Tx for the review.